docker/README.md

```diff
@@ -61,44 +61,44 @@ Start `btcd`, and then create an address for `Alice` that we'll directly mine
 bitcoin into.
 ```shell
 # Init bitcoin network env variable:
-⛰  export NETWORK="simnet" 
+export NETWORK="simnet" 
 
 # Create persistent volumes for alice and bob.
-⛰  docker volume create simnet_lnd_alice
-⛰  docker volume create simnet_lnd_bob
+docker volume create simnet_lnd_alice
+docker volume create simnet_lnd_bob
 
 # Run the "Alice" container and log into it:
-⛰  docker-compose run -d --name alice --volume simnet_lnd_alice:/root/.lnd lnd
-⛰  docker exec -i -t alice bash
+docker-compose run -d --name alice --volume simnet_lnd_alice:/root/.lnd lnd
+docker exec -i -t alice bash
 
 # Generate a new backward compatible nested p2sh address for Alice:
-alice ⛰  lncli --network=simnet newaddress np2wkh 
+alice lncli --network=simnet newaddress np2wkh 
```

Comment:
Will this work?

Reply:
Good point, I thought about this one as well.

Thinking over it again, the intention is to clearly convey that we want to run lncli as either alice or bob user, so usage of $ is predictably appropriate here.

Furthermore, this is consistent with https://dev.lightning.community/tutorial/01-lncli/, another piece of documentation maintained by Lightning Labs.

Updated [doc: append $ to bob/alice lncli commands [skip ci]](https://github.com/lightningnetwork/lnd/pull/7070/commits/423e3700785d0496133acbe25417b7d1a07ba299)

Also I agree with the scripted-diff application, will add that shortly